pkg/apiserver/apiserver.go

```diff
@@ -1,6 +1,8 @@
 package apiserver
 
 import (
+	"github.com/golang/glog"
+
```

Comment:
only if you're in here for some other reason... let's remove these blank lines in the imports.
I think the traditional way is to have one between golang imports and "others" but not between groups of others.
e.g.  https://github.com/kubernetes/kubernetes/blob/master/pkg/api/conversion.go